### ISSUE TITLE
Missing selector in Broker resource for selecting security groups 

### ISSUE THREAD

**Issue by kaushik287** [upbound/provider-aws#726]
What problem are you facing?
While trying to create a Broker resource with security groups using same(one) composition I'm unable to select Security groups using labels as securityGroupIdSelector is missing for Broker resource.

How could Official AWS Provider help solve your problem?
Can you add the missing selector which will be something like **securityGroupIdSelector** to the **Broker** resource. This is already available in the community contributed aws-provider. Please add it in the upbound provider as well

API: mq.aws.upbound.io/v1beta1
Kind: Broker

**Comment by kaushik287** [comment]
Request you to please provide a update on this as it has been a blocker for on going development. Thanks

**Comment by jeanduplessis** [comment]
@kaushik287 Upbound prioritized the needs of its customers in terms of directing development efforts across the official providers it supports. This issue isn't currently a priority. We welcome a PR to address the issue from you or other community members that might be impacted by it.

**Comment by robsclark** [comment]
@jeanduplessis more info on this issue:

After creation of managed resource Security group, to select the Created Security group as security group for Active MQ resource using labels, SecurityGroupIdSelctor parameter is missing in Broker resource. This parameter is already present in Crossplane AWS community provider but by using community provider a race condition is getting triggered by which an annotation is added by Crossplane on Active MQ resource and resource goes to Pending creation state. On force removal of annotation after security group creation, annotation gets readded and state remains same as pending creation

**Comment by ayeks** [comment]
@robsclark @kaushik287 Hey, I ran into the same problem. What is your solution to it?

I managed to get it to work by hardcoding the security groups into the chart:

```yaml
    securityGroups:
      - sg-02cXXXXXXXXXXa6 
```

But this is not great when I create the SG also through crossplane for obvious reasons. I could also think of some script that retrieves and  pushes this value somewhere, but this also sucks. Any proposals to solve this?
Pushing a MR to the upbound provider would be awesome, but I did not dove into the codebase yet.
